src/io/flutter/sdk/FlutterSdkManager.java

```diff
@@ -50,7 +79,7 @@ public void removeListener(@NotNull Listener listener) {
   }
 
   private static boolean isGlobalFlutterSdkSet() {
-    return FlutterSdk.getGlobalFlutterSdk() != null;
+    return FlutterSdk.getGlobalFlutterSdk() != null && FlutterSdkUtil.isFluttering();
```

Comment:
Good catch. I was able to remove the prior checks but it is necessary to have that call here.

I changed the name but I'm not sure it is better.